cluster/supported/aks.go

```diff
@@ -0,0 +1,51 @@
+package supported
+
+import (
+	"github.com/banzaicloud/pipeline/cluster"
+	"github.com/banzaicloud/banzai-types/constants"
+	"github.com/banzaicloud/banzai-types/components"
+)
+
+type AzureInfo struct {
+	BaseFields
+}
+
+// GetType returns cloud type
+func (a *AzureInfo) GetType() string {
```

Comment:
I don't really understand why are these methods? it never uses the AzureInfo

Reply:
The 3 current supported clouds are Google (GoogleInfo), Azure (AzureInfo) and Amazon (AmazonInfo) which queried the supported locations, machine types and Kubernetes versions. These 3 structs helps in that. Here the creation: https://github.com/banzaicloud/pipeline/blob/master/cluster/supported/base.go#L44